### PR TITLE
Updated steps to set MAC address for VF interfaces

### PR DESCRIPTION
**Pull request by sureshmarikkannu**
When enabling VF instances sometime valid MAC address not set for these interfaces. So updated the procedure to ensure the devices are set to valid MAC before deploy UPF in DPDK mode

### CODE REVIEW

**Comment by gab-arrobo** [comment]
Please rebase your PR. Thanks!

**Comment by sureshmarikkannu** [comment]
> Please rebase your PR. Thanks!

done

File: deployment/deploymentupf.rst [hunk 1]
```diff
@@ -79,6 +79,29 @@ step 1: Create VF devices and bind them to the vfio-pci driver
 
     ls -l /sys/class/net/ens801f0/device/virtfn*
 
+- Make sure all the VF devices contains valid MAC address(not all zero's),
```
Comment (gab-arrobo) [inline deployment/deploymentupf.rst:82]:
```suggestion
- Make sure all VF devices contain a valid MAC address (not all zero's),
```

File: deployment/deploymentupf.rst [hunk 2]
```diff
@@ -79,6 +79,29 @@ step 1: Create VF devices and bind them to the vfio-pci driver
...
+
+  .. code-block::
+
+    ip link show ens801f0
+
+  If the MAC shown for VF's are all zero's then retrieve the MAC using the PCI obtained above as follows,
```
Comment (gab-arrobo) [inline deployment/deploymentupf.rst:88]:
```suggestion
  If the MAC shown for the VF's are all zero's, then retrieve the MAC using the PCI obtained above, as follows,
```

File: deployment/deploymentupf.rst [hunk 3]
```diff
@@ -79,6 +79,29 @@ step 1: Create VF devices and bind them to the vfio-pci driver
...
+
+  .. code-block::
+
+    cat /sys/bus/pci/devices/0000\:b1\:01.0/net/ens801f0v0/address
+    00:11:22:33:44:50
+
+    cat /sys/bus/pci/devices/0000\:b1\:01.1/net/ens801f0v1/address
+    00:11:22:33:44:51
+
+  Set the above retrieved MAC address for the VF's using below command,
```
Comment (gab-arrobo) [inline deployment/deploymentupf.rst:98]:
```suggestion
  Then, set the VF's MAC address using the value obtained above
```

**Review by gab-arrobo — CHANGES_REQUESTED** [review]

**Review by gab-arrobo — APPROVED** [review]
+1

**Pull request merged**